### PR TITLE
Update CHANGELOG.asciidoc

### PR DESCRIPTION
**Pull request by karenzone**
Adds known issue to 8.4.0 changelog/release notes. 

**PREVIEW:** https://beats_32944.docs-preview.app.elstc.co/guide/en/beats/libbeat/8.4/release-notes-8.4.0.html

### CODE REVIEW

**Comment by karenzone** [comment]
I know that the changelog is generated.  Hoping that a manual fix outside of the release process will work.  🤞

File: CHANGELOG.asciidoc [hunk 1]
```diff
@@ -29,6 +29,16 @@ https://github.com/elastic/beats/compare/v8.4.0\...v8.4.1[View commits]
 === Beats version 8.4.0
 https://github.com/elastic/beats/compare/v8.3.3\...v8.4.0[View commits]
 
+==== Known Issue 
+
+*Auditbeat* 
+Auditbeat/auditd integration may send malformed data and crash in version 8.4.0. {issue}32818[32818]
+
+Suggested resolution:
+Do not start Auditbeat or auditd integration on Elastic Agent at version 8.4.0, and immediately upgrade to 8.4.1.
```
Comment (debadair) [inline CHANGELOG.asciidoc:38]:
```suggestion
Do not start Auditbeat or auditd integration on Elastic Agent at version 8.4.0. Skip 8.4.0 and upgrade directly to 8.4.1. 
```

**Review by debadair — APPROVED** [review]
I don't think there's any issue with updating this topic manually, but the changes might get trounced when the 8.4.2 changelog is generated.

File: CHANGELOG.asciidoc [hunk 2]
```diff
@@ -29,6 +29,16 @@ https://github.com/elastic/beats/compare/v8.4.0\...v8.4.1[View commits]
 === Beats version 8.4.0
 https://github.com/elastic/beats/compare/v8.3.3\...v8.4.0[View commits]
 
+==== Known Issue 
+
+*Auditbeat* 
+Auditbeat/auditd integration may send malformed data and crash in version 8.4.0. {issue}32818[32818]
```
Comment (karenzone) [inline CHANGELOG.asciidoc:35]:
```suggestion
Auditbeat/auditd integration will send malformed data and may crash in version 8.4.0. {issue}32818[32818]
```

File: CHANGELOG.asciidoc [hunk 3]
```diff
@@ -29,6 +29,16 @@ https://github.com/elastic/beats/compare/v8.4.0\...v8.4.1[View commits]
 === Beats version 8.4.0
 https://github.com/elastic/beats/compare/v8.3.3\...v8.4.0[View commits]
 
+==== Known Issue 
+
+*Auditbeat*
```
[the remark this answers is not in the document]
Comment (karenzone) [reply CHANGELOG.asciidoc:34]: Add return for proper formatting

**Review by ollyhowell — APPROVED** [review]
LGTM!

**Pull request merged**